File: classic puzzle-medium/rubik%C2%AE.cpp
//https://www.codingame.com/training/medium/rubik%C2%AE
//RUBIK®
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    int n;
    cin >> n; cin.ignore();
    long ans=n*n*2+(n*2+2*(n-2))*(n-2);
    if(n==1) ans=1;
    cout << ans << endl;
}

File: classic puzzle-medium/rubik%C2%AE.js
//https://www.codingame.com/training/medium/rubik%C2%AE
//RUBIK®
const N = parseInt(readline())
let ans=N*N*2+((N*2)+2*(N-2))*(N-2)
if(N==1) ans=1
console.log(ans);
